draft-ietf-quic-http.md

```diff
@@ -429,16 +429,66 @@ Frame type 0x3 is reserved.
 
 ### SETTINGS {#frame-settings}
 
-The SETTINGS frame (type=0x04) is unmodified from {{!RFC7540}} (so far). It MUST 
-only be sent on the connection control stream (Stream 3). 
-
-As in HTTP/2, additional SETTINGS frames may be sent mid-connection by either 
-endpoint. 
-
-TODO:
-: Decide whether to acknowledge receipt of SETTINGS through empty SETTINGS
-  frames with ACK bit set, as in HTTP/2, or rely on transport- level
-  acknowledgment.
+The SETTINGS frame (type=0x4) conveys configuration parameters that affect how 
+endpoints communicate, such as preferences and constraints on peer behavior, and 
+is substantially different from {{!RFC7540}}. Individually, a SETTINGS parameter 
+can also be referred to as a "setting". 
+
+SETTINGS parameters are not negotiated; they describe characteristics of the 
+sending peer, which are used by the receiving peer. However, a negotiation can 
+be implied by the use of SETTINGS -- a peer uses SETTINGS to advertise a set of 
+supported values. The recipient can then choose which entries from this list are 
+also acceptable and proceed with the value it has chosen. (This choice could be 
+announced in a field of an extension frame, or in a value in SETTINGS.) 
+
+Different values for the same parameter can be advertised by each peer. For 
+example, a server might support many different signing algorithms, while a 
+resource constrained client has only one or two that it can validate. 
+
+A SETTINGS frame MAY be sent at any time by either endpoint over the lifetime 
+of the connection. 
+
+Each parameter in a SETTINGS frame replaces any existing value for that 
+parameter. Parameters are processed in the order in which they appear, and a 
+receiver of a SETTINGS frame does not need to maintain any state other than the 
+current value of its parameters. Therefore, the value of a SETTINGS parameter is 
+the last value that is seen by a receiver.
+
+The SETTINGS frame defines the following flag: 
+
+  REQUEST_ACK (0x1):
```

Comment:
I think that this will be formatted strangely.  You should confirm with [the manual](http://kramdown.gettalong.org/syntax.html) and maybe check the build output.